input/tail.md

```diff
@@ -17,6 +17,16 @@ It is included in Fluentd's core.
   <parse>
     @type apache2
   </parse>
+  <group>
+    rate_period 30s
+    <rule>
+      match {
+          namespace: /space1|space2|space2/,
+          podname: /app.*/,
+      }
+      limit 200
+    </rule>
+  </group>
```

Comment:
For the same reason, I think it would be better to leave the simple configuration example here as it is
and have a separate configuration example for this feature.

Reply:
How about moving this example to just below the `group` Section and reverting this block?

Reply:
Just removed in c0a9c1e35276d9e33ec8280b6a33f6aa9c46c253
Since another example already exist just under `<group>`, I simply removed it.